packages/sdk/src/schema/animation-schema.ts

```diff
@@ -0,0 +1,207 @@
+import { StyleValue } from "@webstudio-is/css-engine";
+import { z } from "zod";
+
+// Helper for creating union of string literals from array
+const literalUnion = <T extends readonly string[]>(arr: T) =>
+  z.union(
+    arr.map((val) => z.literal(val)) as [
+      z.ZodLiteral<T[0]>,
+      z.ZodLiteral<T[1]>,
+      ...z.ZodLiteral<T[number]>[],
+    ]
+  );
+
+// Range Units
+export const RANGE_UNITS = [
+  "%",
+  "px",
+  "cm",
+  "mm",
+  "q",
+  "in",
+  "pt",
+  "pc",
+  "em",
+  "rem",
+  "ex",
+  "rex",
+  "cap",
+  "rcap",
+  "ch",
+  "rch",
+  "lh",
+  "rlh",
+  "vw",
+  "svw",
+  "lvw",
+  "dvw",
+  "vh",
+  "svh",
+  "lvh",
+  "dvh",
+  "vi",
+  "svi",
+  "lvi",
+  "dvi",
+  "vb",
+  "svb",
+  "lvb",
+  "dvb",
+  "vmin",
+  "svmin",
+  "lvmin",
+  "dvmin",
+  "vmax",
+  "svmax",
+  "lvmax",
+  "dvmax",
+] as const;
+
+export const rangeUnitSchema = literalUnion(RANGE_UNITS);
+
+export const rangeUnitValueSchema = z.union([
+  z.object({
+    type: z.literal("unit"),
+    value: z.number(),
+    unit: rangeUnitSchema,
+  }),
+  z.object({
+    type: z.literal("unparsed"),
+    value: z.string(),
+  }),
+]);
+
+// @todo: Fix Keyframe Styles
+export const keyframeStylesSchema = z.record(StyleValue);
```

Comment:
Would it be possible to not load css-engine in component runtime?

Reply:
toValue is used

Reply:
Will fix in next PRs